contracts/incentives/DistributionManager.sol

```diff
@@ -3,9 +3,8 @@ pragma solidity 0.7.5;
 pragma experimental ABIEncoderV2;
 
 import {IAaveDistributionManager} from '../interfaces/IAaveDistributionManager.sol';
-import {SafeMath} from '../lib/SafeMath.sol';
 import {DistributionTypes} from '../lib/DistributionTypes.sol';
-
+import '@aave/aave-stake/contracts/lib/SafeMath.sol';
```

Comment:
I believe there was a collision when trying to verify between the two different SafeMath libraries so I changed the import here